src/components/photoGallery/PhotoGallery.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from "react";
+import "./PhotoGallery.scss"
+
+export const PhotoGallery: React.FunctionComponent = () => {
+
+  const [imageUrls, setImageUrls] = useState<string[]>();
+  const img_urls: string[] = [];
+
+        useEffect(() => {
+          const url = `https://api.nasa.gov/planetary/apod?count=50&api_key=DEMO_KEY`;
+          
+          const fetchData = async () => {
+            try {
+              const response = await fetch(url);
+              const json = await response.json();
+              json.forEach((item: {url: string}) => img_urls.push(item.url));  
+              setImageUrls(img_urls);
+            } catch (error) {
+              console.log("error", error);
```

Comment:
This isn't a very helpful error message! If you can't add any useful information, you are actually _making it worse_ by catching this error. Please either remove this `try/catch` block, or handle the error properly!